utt/wallet-cli/include/wallet.hpp

```diff
@@ -81,9 +81,7 @@ class Wallet {
   /// @brief Updates the ERC20 balance of the wallet.
   void updatePublicBalance(Channel& chan);
 
-  struct DummyUserStorage : public utt::client::IUserStorage {};
-
-  DummyUserStorage storage_;
+  std::unique_ptr<utt::client::IStorage> storage_;
```

Comment:
Also, as discussed offline - let's run the automated E2E test against a build containing these changes. Please add the outcome of that run in the "Testing done" section.